Fix to avoid a bug when there are no changes in commit list

peergum opened the pull request:
Also removed a duplicate import

samrocketman commented:
Is this related to #30 ? When you encounter the bug?  Just looking where I can test it.

peergum commented:
If you start a build manually and there has been no change since last build, it will complete (successfully or not doesn't matter anymore now that the commit list can be shown in all cases) and the commit list should be shown, but the null value returned was creating a null pointer exception. I patched it a few days after my pull request in november, but forgot to sync it.
